src/main/java/io/jenkins/plugins/sample/global_configuration/OnboardingPluginGlobalConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package io.jenkins.plugins.sample.global_configuration;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import org.jenkinsci.Symbol;
+
+@Extension
+@Symbol("OnboardingPlugin")
+public class OnboardingPluginGlobalConfiguration extends GlobalConfiguration {
```

Comment:
Values from the form won't be stored. You need to create the fields with the setter and getter

Reply:
Sure @fcojfernandez. I thought #level1 is just about the UX hence not bother about it. But thanks for highlighting :)

Reply:
Added setter/getter. If I understand correctly setter/getter will get called on the 'Save' action.

Reply:
> Sure @fcojfernandez. I thought #level1 is just about the UX hence not bother about it. But thanks for highlighting :)

it might be. This exercise is so new that I might have missundertood the scope of each level. If that the case, just take my next comment/explanation as educative in general and nothing for this PR

> Added setter/getter. If I understand correctly setter/getter will get called on the 'Save' action.

Explained in a light way so it's more understandable, in Jenkins you have `Describable` and `Descriptor` classes. The former are the classes to model your configuration or your item. The latter are in charge of handle the form, creating the Describable class and the adding the form validations.
Besides, `staple` is the library behind that.

To create the object behind the form you nee:
* Constructor annotated as `DataBoundConstructor` and every parameter in that constructor needs a getter method. If it doesn't exist, then it must exist the constructor without parameter
* Properties not in the `DataBoundConstructor`, but then they need a getter and a setter annotated as `DataBoundSetter`

With these annotations then the values from the form are bind in the java object. So you are missing either the DataBoundConstructor or the DataBoundSetter annotation